src/app/containers/JournalpostLoader.tsx

```diff
@@ -55,6 +53,23 @@ export const JournalpostLoaderImpl: React.FunctionComponent<JournapostLoaderProp
 }) => {
     const { journalpostid } = useParams<{ journalpostid: string }>();
 
+    const [pendinglukkDebuggJp, setPendinglukkDebuggJp] = useState(false);
+    const [lukkDebuggJpStatus, setLukkDebuggJpStatus] = useState<number | undefined>(undefined);
+    const [ingenJp, setIngenJp] = useState(false);
```

Comment:
er denne nødvendig? Hvis journalposten ikke finnes får vi vel aldri conflict, bare 404

Reply:
Disse brukes for å håndtere svar fra /lukkDebugg/... Den svarer 200, 400, 404 eller 500 f.eks. Jeg bruker det for å vise svar til bruker fra apet etter at bruker trykker knapp - lukk jogrnalpost